core/sphinx/surb.go

```diff
@@ -29,9 +29,7 @@ const (
 	sprpKeyMaterialLength = crypto.SPRPKeyLength + crypto.SPRPIVLength
 )
 
-// NewSURB creates a new SURB with the provided path using the provided entropy
-// source, and returns the SURB and decrypion keys.
-func (s *Sphinx) NewSURB(r io.Reader, path []*PathHop) ([]byte, []byte, error) {
+func (s *Sphinx) newNikeSURB(r io.Reader, path []*PathHop) ([]byte, []byte, error) {
```

Comment:
we should keep this docstring too

Reply:
We do keep it in the public method.

Reply:
Please do and see new_sphinx.go where we have:
```
// NewSURB creates a new SURB with the provided path using the provided entropy
// source, and returns the SURB and decrypion keys.
func (s *Sphinx) NewSURB(r io.Reader, path []*PathHop) ([]byte, []byte, error) {

```